### ISSUE TITLE
Using RangeDailyBase with dummy task fails with "Unfulfilled dependencies"

### ISSUE THREAD

**Issue by robbles** [spotify/luigi#1093]
When trying to use the RangeDailyBase class as described in [Triggering recurring tasks](http://luigi.readthedocs.org/en/stable/luigi_patterns.html#triggering-recurring-tasks) with a dummy task that triggers several tasks with a date parameter, it fails at the end of a run with the following error:

```
RuntimeError: Unfulfilled dependencies at run time: UpdateRedshiftTables(date=2015-07-23), UpdateRedshiftTables(date=2015-07-24), UpdateRedshiftTables(date=2015-07-25)
```

The command I used is:

```
luigi --module update_redshift --local-scheduler RangeDailyBase --of UpdateRedshiftTables --start 2015-07-23
```

The dummy task looks like this (pretty much the exact same setup as in the docs):

```python
class UpdateRedshiftTables(luigi.Task):
    date = luigi.DateParameter()

    def requires(self):
        yield UpdateRedshiftTableA(date=self.date)
        yield UpdateRedshiftTableB(date=self.date)
```

It looks to me like RangeDailyBase expects the task it runs to have an output. Is there a workaround for this? Do I need to give `UpdateRedshiftTables` a dummy output to the filesystem for this to work?

Also: I've checked this with the individual tasks (which have outputs), and they complete successfully with no errors when run with RangeDailyBase. I can also run the `UpdateRedshiftTables` task directly for a single date with no errors.

**Comment by Tarrasch** [comment]
Shouldn't `UpdateRedshiftTables` be a wrapper task? I mean like

```
class UpdateRedshiftTables(luigi.WrapperTask):
    ...
```

**Comment by robbles** [comment]
@Tarrasch Yeah, I realized that yesterday and it fixed the problem.

The docs on the page I referenced make no mention of WrapperTask though. There's actually no mention of it in the docs anywhere other than the API docs. It should be used at least in the "Triggering Many Tasks" section, as the code sample there is effectively broken.

**Comment by Tarrasch** [comment]
@robbles, cool. Would you like to contribute back by fixing the docs? pretty please? here's a cute dog. :dog:

**Comment by robbles** [comment]
@Tarrasch sure, no problem :)

**Comment by Tarrasch** [comment]
Cool! Fixed in #1097
